operator/poolboy_templating.py

```diff
@@ -81,6 +81,25 @@ def seconds_to_interval(seconds:int) -> str:
     else:
         return f"{int(seconds)}s"
 
+def timedelta_to_str(td:timedelta) -> str:
+    ret = ""
```

Comment:
@jkupferer wdt?

    total_seconds = int(td.total_seconds())
    days = total_seconds // 86400
    hours = (total_seconds % 86400) // 3600
    minutes = (total_seconds % 3600) // 60
    seconds = total_seconds % 60

Reply:
Yeah, that looks better to me too :smile: